packages/accounts/src/msca/account/multisigAccount.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  createBundlerClient,
+  getAccountAddress,
+  getVersion060EntryPoint,
+  toSmartContractAccount,
+  type Address,
+  type EntryPointDef,
+  type SmartAccountSigner,
+  type SmartContractAccount,
+  type SmartContractAccountWithSigner,
+  type UserOperationRequest,
+} from "@alchemy/aa-core";
+import {
+  concatHex,
+  encodeFunctionData,
+  hexToBigInt,
+  type Chain,
+  type Hex,
+  type Transport,
+} from "viem";
+import { MultisigModularAccountFactoryAbi } from "../abis/MultisigModularAccountFactory.js";
+import { multisigMessageSigner } from "../plugins/multisig/signer.js";
+import { getDefaultMultisigModularAccountFactoryAddress } from "../utils.js";
+import { standardExecutor } from "./standardExecutor.js";
+
+export type MultisigModularAccount<
+  TSigner extends SmartAccountSigner = SmartAccountSigner
+> = SmartContractAccountWithSigner<"MultisigModularAccount", TSigner> & {
```

Comment:
```suggestion
> = SmartContractAccountWithSigner<MULTISIG_ACCOUNT_SOURCE, TSigner> & {
```

also can we bring the `MULTISIG_ACCOUNT_SOURCE` up to the top of this file

Reply:
Done, but noting that I had to use `typeof MULTISIG_ACCOUNT_SOURCE` instead.